doc/how-to-run-in-windows.rst

```diff
@@ -0,0 +1,142 @@
+How to use Cabal in Windows
+===========================
+
+This document describes how to use Cabal in a Windows system. See the
+:ref:`Further reading` section for some other references that might provide some
+more explanations. For a TL;DR, jump to the :ref:`Complete configuration`.
+
+Install the Haskell environment
+-------------------------------
+
+The recommended way of setting up a Haskell environment in Windows is by using
+`GHCup <https://www.haskell.org/ghcup/>`_. Follow the steps outlined in its
+webpage to install at least GHC and Cabal. GHCup will install its own MSYS2
+system in your computer unless told not to do so, refer to `its documentation
+<https://www.haskell.org/ghcup/install/#windows_1>`_ for more information.
+
+Ensure that Cabal can call UNIX-like tools
+------------------------------------------
+
+As Cabal needs sometimes to call UNIX-like tools that come with MSYS2 (such as
+``make`` or even ``git``), the directories where those are located need to be
+made visible in the ``PATH``. For that, Cabal provides the ``extra-prog-path``
+configuration option. Your :ref:`global configuration <config-file-discovery>`
+should include this option:
+
+::
+
+   extra-prog-path: <msys-dir>\usr\bin
+                    <msys-dir>\<environment>\bin

```

Comment:
The order is wrong. This can cause problems.

```suggestion
   extra-prog-path: <msys-dir>\<environment>\bin
                    <msys-dir>\usr\bin

```

Reply:
Done, thanks!